pyxem/generators/calibration_generator.py

```diff
@@ -45,6 +46,11 @@ class CalibrationGenerator:
 
     """
 
+    @deprecated(
+        since="0.18.0",
+        removal="1.0.0",
+        alternative="pyxem.signals.Diffraction2D.calibrate",
+    )
```

Comment:
This doesn't need to be changed, but it's worth pointing out that we should consider making a deprecated class decorator in future because currently this says that `__init__` is deprecated.